ansible_base/rest_filters/rest_framework/field_lookup_backend.py

```diff
@@ -22,8 +22,6 @@ class FieldLookupBackend(BaseFilterBackend):
     Filter using field lookups provided via query string parameters.
     """
 
-    RESERVED_NAMES = ('page', 'page_size', 'format', 'order', 'order_by', 'search', 'type', 'host_filter', 'count_disabled', 'no_truncate', 'limit', 'validate')
```

Comment:
@AlanCoding Isn't this list also preventing users from using these column names in their database model. If they have a "type" column in their DB then they cannot filter on it.

Reply:
Should there be a warning if the column name also shows up in the reserved list.

Reply:
I don't think a warning with the request would be the right tool for this. The Django checks framework might be a better fit

https://docs.djangoproject.com/en/5.0/topics/checks/